Mailbox/Size.cs

```diff
@@ -1,4 +1,22 @@
-﻿namespace Mailbox
-{
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
 
+namespace Mailbox
+{
+    public static class SizeExt
```

Comment:
In general you want to avoid abbreviations for identifier names.
```suggestion
    public static class SizeExtensions
```